features/step_definitions/bookmark_steps.rb

```diff
@@ -310,6 +310,18 @@
   step %{all indexing jobs have been run}
 end
 
+Given "{string} has a bookmark of a work titled {string}" do |user, title|
+  step %{the user "#{user}" exists and is activated}
+
+  user_pseud = User.find_by(login: user).default_pseud
+  work1 = FactoryBot.create(:work, title: title)
+  FactoryBot.create(:bookmark,
+                    bookmarkable_id: work1.id,
+                    pseud_id: user_pseud.id)
```

Comment:
Nitpick: `FactoryBot` can automatically get the ID we just pass the object, which is IMO a bit clearer
```suggestion
                    bookmarkable: work1,
                    pseud: user_pseud)
```

Reply:
Thanks, this is now done.